ui/src/app/base/components/node/HardwareCard/HardwareCard.test.tsx

```diff
@@ -0,0 +1,54 @@
+import { mount } from "enzyme";
```

Comment:
This file is unchanged, apart from the new name of the component (`SystemCard` to `HardwareCard`) and props API (`systemId` to `node`)